cellprofiler/modules/dilateobjects.py

```diff
@@ -0,0 +1,87 @@
+# coding=utf-8
+
+"""
+DilateObjects
+===================
```

Comment:
reST will complain if the `===`s isn't the same length as the line above. Please adjust the length of this line appropriately.